packages/jest-haste-map/src/types.js

```diff
@@ -23,3 +24,11 @@ export type WorkerMetadata = {
   dependencies: ?Array<string>,
 };
 export type WorkerCallback = (error: ?Error, metaData: ?WorkerMetadata) => void;
+
+export type CrawlerOptions = {
```

Comment:
Exact types are awesome and are the new default for me when I don't expect the object to take on any more fields than I describe. It guards against typos and over-specified objects. Basically an exact type means that the object must have all the specified fields but cannot have more fields than that. For most APIs, being this restrictive is a great thing. Personally, it helps me move a lot faster on Jest: when I want to add a field, I add it to the type and then Flow tells me exactly where I need to add the field + if I mistype it, flow will yell at me.